filament/CMakeLists.txt

```diff
@@ -198,7 +198,10 @@ set(MATERIAL_SRCS
         src/materials/skybox.mat
         src/materials/ssao/sao.mat
         src/materials/ssao/saoBentNormals.mat
-        src/materials/separableGaussianBlur.mat
+        src/materials/separableGaussianBlur1.mat
```

Comment:
Did you test the impact on binary size before/after?

Reply:
no, but do we even have some guidance about that? Generally we have prioritized performance.

Reply:
It depends on what the increase is. If it adds 100 KiB to a single target, I'd say it's not worth it.